clang/test/dpct/device_usmnone.cu

```diff
@@ -0,0 +1,12 @@
+// RUN: dpct --format-range=none -out-root %T/device_usmnone %s --cuda-include-path="%cuda-path/include" --usm-level=none -- -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/device_usmnone/device_usmnone.dp.cpp
+#include <cuda.h>
+
+int main() {
+int concurrentManagedAccess = 0;
+int p_gpuDevice = 0;
+// CHECK: /*
+// CHECK-NEXT:  DPCT1076:{{[0-9]+}}: The device attribute was not recognized. You may need to adjust the code.
```

Comment:
DPCT1076 is not for this scenario, suggest return a false value if USM-none doesn't support concurrent managerd access.